ner_v2/detectors/numeral/number_range/README.md

```diff
@@ -0,0 +1,246 @@
+  # language_script='en'
+  
+  $ URL='localhost'
+  $ PORT=8081
+  
+  $ curl -i 'http://'$URL':'$PORT'/v2/number_range?message=do%20hajaar%20char%20sau&entity_name=number&structured_value=&fallback_value=&bot_message=&source_language=en&language_script=en&unit_type='
+  
+  # Curl output
+  $ {
+      "data": [
+          {
+              "detection": "message",
+              "original_text": "more than 12",
+              "entity_value": {
+                  "max_value": null,
+                  "min_value": "12",
+                  "unit": null
+              },
+              "language": "en"
+          }
+      ]
+  }
+  ```
+
+  
+
+### Steps to add new language for Number Range detection
+
+In order to add any new language you have to follow below steps:
+
+1. Create a directory with `ISO 639-1` code of that language inside `ner_v2/detectors/numeral/number_range/`.  
+
+2. Create a directory named `data` inside the language_code folder.
+
+3. Add a CSV file named `number_range_keywords.csv` inside data folder.  
+
+   Below is the folder structure of same after adding all the files for new language `xy`.
+
+   ```python
+   |__ner_v2
+         |___detectors
+             |___numeral
+                 |___number_range
+                     |___xy    # <- New language Added 
+                     |	  |___data
+                     |      |___number_range_keywords.csv
+                     |      
+                     |__number_detection.py 
+   ```
+
+
+
+
+####  GuideLines to create data files
+
+Below is the brief about how to create data file `number_range_keywords.csv` All the description of the file is explained using Hindi as a reference language. 
+
+1. **number_range_keywords.csv**:  This file contains the vocabs for range keywords, their position around number values and their range type.
+
+   |                        range_variants                        | position | range_type |
+   | :----------------------------------------------------------: | :------: | :--------: |
+   | k uper\| se upar\| se jada \| se adhik \| के ऊपर\|  से ज्यादा \|से जादा \| से अधिक |    1     |    min     |
+   |                     kam se kam\| कम से कम                     |    -1    |    min     |
+   | jyada se jyada \| lagbhag \| ज्यादा से ज्यादा \| जादा से जादा \| लगभग |    -1    |    max     |
+   | se niche \| se kam \| se sasta \| se saste k aas paas\|  k aas pas\| k lagbhag \|  से नीचे \| से कम \| से सस्ता \| से सस्ते \|   के आस पास  \| के लगभग |    1     |    max     |
+   |                           se\|-\|से                           |    0     |  min_max   |
+
+   Here, the 1st column will contain the variants of range keywords separated by pipes, which are present before or after or in between number values and defines what type of number value is present, either min or max. 
+
+   2rd column corresponds to the position of the keyword. If 1 means it will be present after number value, -1 if present before number.  
```

Comment:
`2rd` -> `2nd`

`corresponds to the position of the keyword. If 1 means it will be present after number value, -1 if present before number. ` ->

```
indicates the position of the variants in the 1st column around a number in the text.

- -1 means the variant appears as a prefix to the number in the target language (E.g. `kam se kam` 500 (at least 500))
- 0 means the variant appears sandwiched between two numbers in the target language (E.g. `300 se 500` (300 to 500))
- 1 means the variant appears as a suffix to the number in the target language (E.g. `500 se niche` (less than 500))
```